center/src/main/resources/application.yml

```diff
@@ -54,14 +54,16 @@ app:
   storage:
     type: ${STORAGE_TYPE:AZURE} # current available options: LOCAL, AZURE
     blob:
-      connection: ${BLOB_CONNECTION_STR}
+      connection: ${BLOB_CONNECTION_STR:}
       fileLimitDay: 6
       CDNUrl: ${CDN_URL:}
       SASExpiryTimeFront: ${BLOB_SAS_EXPIRY_FRONT:120}
       SASExpiryTimeAgent: ${BLOB_SAS_EXPIRY_AGENT:120}
       SASExpiryUpdate: ${BLOB_SAS_EXPIRY_UPDATE:10}
       timeUnit: ${TIME_UNIT:MINUTES}
-#    local:
+    local:
+      token: ${CENTER_TOKEN:token=CENTER_LOCAL_STORAGE_TOKEN}
+      endpoint: ${LOCAL_STORAGE_ENDPOINT:http://localhost:9886/}
```

Comment:
Shouldn't this follow the center host config value?

Reply:
We may not need a endpoint for local storage, or by default this follow center host.

Reply:
We still need to pass in a value, as Center itself cannot specify the host to send the download/upload requests as no other config value is able to be leveraged here.
We can then leverage this for Agent directly to make sure if there's other local usage enabled scenario that involves Agent and Center are on different physical machines within a LAN, such as some scenario similar to the test environment.